Services/Models/Simulation.cs

```diff
@@ -11,18 +11,20 @@ public class Simulation
     {
         private DateTimeOffset? startTime;
         private DateTimeOffset? endTime;
+        private DateTimeOffset? stopTime;
         private string iotHubConnectionString;
 
         [JsonIgnore]
         public string ETag { get; set; }
 
-        [JsonIgnore]
```

Comment:
as discussed offline, undo this change. Add comment explaining why we use JsonIgnore, smth like:  "when written to storage, Id and Etag are not serialized, and are instead written in dedicated columns..."